### PR TITLE
update version

### PR DESCRIPTION
**Pull request by dnaumenko**

### CODE REVIEW

**Comment by dnaumenko** [comment]
@kaishh , @pshirshov , Server/Client compiles just fine ok with pull request updates, but clients fails with 500 error, and Server has this stacktrace:

```
E 2019-07-21T22:29:01.595+03:00[Europe/Kiev] c.g.p.i.i.r.r.h.HttpServer.handleResult ...776530151:37  (HttpServer.scala:290) context=com.github.pshirshov.izumi.idealingua.runtime.rpc.http4s.HttpRequestContext@37e942cc: Execution failed, termination, method=LoginService.signUp, request=AuthedRequest(AdminRequest(CompanyId#1,NetworkContext(127.0.0.1,UNKNOWN)),Request(method=POST, uri=/v2/LoginService/signUp, headers=Headers(Host: localhost:8080, Authorization: <REDACTED>, User-Agent: http4s-blaze/0.20.0-M5, Content-Length: 140))), cause=java.lang.NoSuchMethodError: com.github.pshirshov.izumi.functional.bio.BIO.pure(Ljava/lang/Object;)Ljava/lang/Object;
java.lang.NoSuchMethodError: com.github.pshirshov.izumi.functional.bio.BIO.pure(Ljava/lang/Object;)Ljava/lang/Object;
	at com.septimalmind.services.auth.LoginServiceWrappedServer$signUp$.$anonfun$invoke$4(LoginService.scala:118)
	at com.septimalmind.services.auth.LoginServiceWrappedServer$signUp$.$anonfun$invoke$4$adapted(LoginService.scala:118)
	at scalaz.zio.IO$Redeem.apply(IO.scala:909)
	at scalaz.zio.IO$Redeem.apply(IO.scala:900)
	at scalaz.zio.internal.FiberContext.nextInstr(FiberContext.scala:417)
	at scalaz.zio.internal.FiberContext.evaluateNow(FiberContext.scala:173)
	at scalaz.zio.internal.FiberContext.$anonfun$evaluateLater$1(FiberContext.scala:312)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by neko-kai** [comment]
@dnaumenko did you update izumi version in the project itself? Versions should match exactly, izumi does not do bincompat

**Comment by dnaumenko** [comment]
@kaishh  let me check

**Comment by dnaumenko** [comment]
@kaishh , updated a server, but it fails to compile now :/ Seems like Server can't derive Cats instances for ConcurrentEffect, plus there are other breaking changes. 

Will try to fix them tomorrow

**Comment by neko-kai** [comment]
@dnaumenko ZIO ConcurrentEffect needs implicit `zio.Runtime` to be available, try using `ZIO.runtime.flatMap { implicit runtime => <code> }` or add `implicit runtime: Runtime[Any]` as a parameter...

**Comment by dnaumenko** [comment]
@kaishh Ok, both Client and Server are running just fine with latest stable version for now. Please, check code. Not sure if it's worth  to make it any better since we agreed on getting back to samples later when next version of IDL is ready.

File: server/build.sbt [hunk 1]
```diff
@@ -23,8 +23,8 @@ libraryDependencies := Seq(
   "io.7mind.izumi"                %% "idealingua-v1-compiler"           % "0.8.6",
   "io.7mind.izumi"                %% "idealingua-v1-compiler"           % "0.8.6",
   "io.7mind.izumi"                %% "idealingua-v1-runtime-rpc-http4s" % "0.8.6",
-  "dev.zio"                       %% "zio"                              % "1.0.0-RC10-1",
```
Comment (neko-kai) [inline server/build.sbt]: You can update to izumi `0.8.7` and bump zio / http4s versions back up to latest

**Comment by neko-kai** [comment]
@dnaumenko Do you want to merge now?

File: server/build.sbt (hunk 1 above)
Comment (dnaumenko) [reply server/build.sbt]: Ah, cool. I will update to `0.8.7`

**Comment by dnaumenko** [comment]
@ Actually, let's merge it as is

**Pull request merged**